src/auth/AuthContext.tsx

```diff
@@ -89,6 +97,7 @@ const authReducer: AuthReducer = (prevState, action): AuthReducerState => {
         customerNumber,
         authStatus,
       });
+      idTokenGlobal = action.idToken.token;
```

Comment:
Since `idToken` in the state here is of type `FirebaseAuthTypes.IdTokenResult`, the string name for the token on idToken should probably be different?

I guess it is really `idTokenTokenGlobal` atm? 😛

Reply:
Another option is to assign just `action.idToken` here, and then use `getIdTokenGlobal().token` in client.ts

Reply:
I renamed the state field and action field to `idTokenResult`, since that was the most precise solution.